src/blockdb/wrapper.h

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) 2009-2010 Satoshi Nakamoto
```

Comment:
Wrapper is overused, and it doesn't explain the most important thing, which is what's being wrapped.

Let's call this file "blockstorage.h", unless you have a better idea.

Also, consider blockdb.* -> blockleveldb.*  The naming now indicates that its a particular implementation of the block database on top of leveldb.  This extends naturally.  For example, we could add blocksqldb.* if we wanted to add a sql database.